packages/luals-annotations/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import { classes, functions, matchers } from "./json-docs";
+import AnnotationBuilder from "./annotationBuilder";
+import * as fs from "fs";
+
+const document = new AnnotationBuilder();
+document.add("---@meta");
+
+classes.forEach((classDef) => {
+	document.withClass(classDef.name, classDef.description, false, () => {
+		classDef.fields?.forEach((field) => document.addField(field));
+		document.addClassDeclaration();
+		classDef.methods?.forEach((method) => document.addFunction(method));
+	});
+});
+
+functions.forEach((func) => {
+	document.addFunction(func);
+});
+
+document.withClass("lest.Matchers", "Matchers for expect()", true, () => {
+	document.addField({
+		name: "never",
+		type: "lest.InverseMatchers",
+		description: "Inverse matchers",
+	});
+
+	document.addClassDeclaration();
+	matchers.forEach((matcher) => document.addFunction(matcher));
+});
+
+document.withClass("lest.InverseMatchers", "Inverse matchers", true, () => {
+	document.addClassDeclaration();
+	matchers.forEach((matcher) => document.addFunction(matcher));
+});
+
+if (process.env["npm_config_debug"]) {
+	console.log(document.toString());
+	process.exit(0);
+}
+
+const targetFilePath = process.argv[2];
+if (!targetFilePath) {
+	console.error("No target file path provided");
+	process.exit(1);
+}
+
+fs.writeFileSync(targetFilePath, document.toString());
```

Comment:
I don't really like `toString` here. It looks like it just calls `.build()` so can we not call that here instead?